src/main/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -282,12 +283,8 @@ public void remove() {
     @Override
     public void close() {
         records.clear();
-        //noinspection EmptyCatchBlock
-        try {
-            if (countingInputStream != null)
-                countingInputStream.close();
-        } catch (final IOException e) {
-        }
+        CloserUtil.close(countingInputStream);
+        CloserUtil.close(referenceSource);
```

Comment:
@tomwhite We're worried that closing this will cause problems for people opening multiple iterators.  I think we should remove that close.